tests/e2e-playwright/specs/02_validate-start-configuration.spec.js

```diff
@@ -1,25 +1,17 @@
 /**
  * WordPress dependencies
  */
-const { test, expect } = require('@wordpress/e2e-test-utils-playwright');
-
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+const { selectors } = require( '../utils/selectors' );
+const { CommonFunction } = require( '../page/Functions.js' )
 test.describe('Validate Start options are visible', () => {
-    test.beforeEach(async ({ admin }) => {
+    
+    test('Check Start Configuration', async ({ admin, page }) => {
         await admin.visitAdminPage('tools.php');
```

Comment:
@alvitazwar In every file you are navigating to the tools page with the help of `visitAdminPage()`  instead you can add that in `NavigateToBloggerPage()` function.